src/instr.c

```diff
@@ -1,7 +1,14 @@
+#include <math.h>
+#include <string.h>
 #include "defs.h"
 #include "err_msg.h"
 #include "instr.h"
-#include <math.h>
+
+#ifndef M_PI
+#define M_PI		3.14159265358979323846
```

Comment:
c99 C doesn't have M_PI defined as a macro, so you'll need to define it yourself.